### PR TITLE
feat(config): add more rules

### PR DESCRIPTION
**Pull request by anehx**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,15 +2,26 @@ module.exports = {
...
+    eqeqeq: "error",
+    "no-alert": "error",
+    "no-else-return": "error",
+    "no-eval": "error",
+    "no-extend-native": "error",
+    "no-extra-bind": "error",
+    "no-floating-decimal": "error",
+
+    // ES6
+    "no-var": "error",
```
Comment (czosel) [inline index.js]: Let's also add the `prefer const over let` rule

**Comment by fkm** [comment]
I'll look into that later. There's still some issue with a deployment I have to look after first.

**Pull request merged**